packages/sysutils/eventlircd/udev.d/98-eventlircd.rules

```diff
@@ -64,6 +64,14 @@ SUBSYSTEMS=="acpi", ATTRS{hid}=="LNXPWRBN", \
   ENV{eventlircd_enable}="true", \
   ENV{eventlircd_evmap}="power.evmap"
 
+SUBSYSTEMS=="input", ATTRS{name}=="key_input", \
```

Comment:
gotcha! this one fixes the hardware power button not working. you can push it directly to master ;)